extensions/2.0/Vendor/EXT_feature_metadata/schema/primitive.EXT_feature_metadata.schema.json

```diff
@@ -24,7 +24,7 @@
       "type": "array",
       "description": "An array of IDs of feature textures from the root `EXT_feature_metadata` object.",
       "items": {
-        "type": "string"
+        "allOf": [ {"$ref": "glTFid.schema.json" } ]
```

Comment:
ohh I see, https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/glTFid.schema.json is essentially just an enum to say "unsigned integer"?

This is a fine change.  Though this is one other thing that makes using `wetzel` difficult for extensions, it doesn't know where to look for this schema file. I think when I tried this in a past iteration of these extensions I had to have a script copy files around for this purpose. not fun.

Reply:
I'm passing it a search path that includes the glTF core schema directory, which seems to avoid the need for copying individual files like `textureInfo.schema.json` and `glTFId.schema.json` around:

```
IGNORE='["featureIdAttribute.featureIds.schema.json", "textureInfo.schema.json", "glTFid.schema.json"]'
SPEC_ROOT=$(dirname "$0")
SEARCH_PATH=$SPEC_ROOT/../../../../specification/2.0/schema

IN_FILE=$SPEC_ROOT/schema/gltf.EXT_feature_metadata.schema.json
wetzel -l 2 -s $SEARCH_PATH -i "$IGNORE" $IN_FILE | replace-between --target $SPEC_ROOT/README.md --token ROOT
```